Assets/PlayModeTests/PatrollingAlgorithmTests/PatrollingAlgorithmTest.cs

```diff
@@ -94,71 +95,71 @@ public IEnumerator Test_ConscientiousReactive_CaveMap()
         {
             var simulation = EnqueueCaveMapScenario(new ConscientiousReactiveAlgorithm());
             _maes.SimulationManager.AttemptSetPlayState(Maes.UI.SimulationPlayState.FastAsPossible);
-            while (!simulation.HasFinishedSim())
+            while (!simulation.HasFinishedSim() && simulation.SimulatedLogicTicks < MaxSimulatedLogicTicks)
             {
                 yield return null;
             }
-            Assert.True(simulation.HasFinishedSim());
+            Assert.True(simulation.HasFinishedSim(), "Simulation did not finish under 25k ticks, indicating the robot is stuck.");
```

Comment:
Nitpick: hardcoding 25k here. Maybe it should use the max ticks variable

Reply:
Done.
God skiferie?